reviews/templates/reviews/documents_list.html

```diff
@@ -26,11 +26,19 @@ <h5>
         </a>
         {% if item.comparable %}
         {% simple_compare_link item.object item.field %}
-        {% endif %}
+        {% endif %} 
     </li>
     {% endfor %}
 </ul>
 
+{% if container.dmp_edit_link %}
```

Comment:
I don't think this is the correct place for this. A proposal only has one data management plan, but if a proposal has four studies the DMP edit link now shows up four times.